File: trunk/server/src/main/java/org/ngame/protocol/auth/AuthResponse.java
/**
 * 登陆返回
 */
package org.ngame.protocol.auth;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import lombok.Data;
import org.ngame.protocol.BaseResponse;
import org.ngame.protocol.annotation.Proto;

/**
 *
 * @author beykery
 */
@Data
@Proto(value = -6, description = "登陆返回")
public class AuthResponse extends BaseResponse
{

  @Protobuf(description = "用户的id==null")
  private String pid;
}

File: trunk/unityClientSample/Assets/src/protocol/AuthResponse.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 登陆返回 协议:-6
    /// </summary>
	[Proto(value=-6,description="登陆返回")]
	[ProtoContract]
	public class AuthResponse
	{
        /// <summary>
        ///  用户的id==null
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public string pid;
        /// <summary>
        ///  是否成功
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public bool success;
        /// <summary>
        ///  错误消息
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string info;

	}
}